### PR TITLE
Step 2: Portfolio accessibility

### PR DESCRIPTION
**Pull request by leehaney254**
## Portfolio accessibility :muscle: 
In this step I:
- Modified the title
- modified the image text alternative
- Modified the headings
- color contrast - No issues found
- Resize - No issues found
-  Modified interactions
- Moving content - No issues found
- Multi-media - No issues found
- Basic structure - No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="styles.css" />
-    <title>Leehaney</title>
+    <title>Leehaney Portfolio</title>
```
Comment (leehaney254) [inline index.html:22]: Modified the title to make it describe the page more and be different from other pages. :100:

File: index.html [hunk 2]
```diff
@@ -311,31 +311,33 @@ <h2 id="heading">About me</h2>
...
           /></a>
         </li>
         <li>
           <a
             class="icons"
             href="https://www.linkedin.com/in/leehaney-george-0a4a51178/"
-            ><img src="./assets/linkedinfoot.png" alt="Linkedin icon"
+            ><img
+              src="./assets/linkedinfoot.png"
+              alt="Link to my linkedin page"
```
Comment (leehaney254) [inline index.html:329]: Modified the image alt text to be more descriptive and null for decoration. :100:

File: index.html [hunk 3]
```diff
@@ -311,31 +311,33 @@ <h2 id="heading">About me</h2>
...
         <li>
           <a class="icons" href="https://twitter.com/Lee06785586"
-            ><img src="./assets/twitterfoot.png" alt="Twitter icon"
+            ><img src="./assets/twitterfoot.png" alt="Link to my twitter page"
           /></a>
         </li>
         <li>
           <a class="icons" href="georgeleehaney@gmail.com"
-            ><img src="./assets/gmailfoot.png" alt="gmail icon"
+            ><img src="./assets/gmailfoot.png" alt="link to my gmail page"
```
Comment (leehaney254) [inline index.html:346]: Modified the image alt text to be more descriptive and null for decoration. :100:

File: index.html [hunk 4]
```diff
@@ -57,31 +57,31 @@ <h1 id="primary">Hey There. <br />I'm Lee</h1>
...
         <li>
           <a class="icons" href="https://twitter.com/Lee06785586"
-            ><img src="./assets/twitter.png" alt="Twitter icon"
+            ><img src="./assets/twitter.png" alt="Linkto my twitter page"
           /></a>
         </li>
         <li>
           <a class="icons" href="georgeleehaney@gmail.com"
-            ><img src="./assets/gmail.png" alt="gmail icon"
+            ><img src="./assets/gmail.png" alt="Link to my gmail page"
```
Comment (leehaney254) [inline index.html:84]: Modified the image alt text to be more descriptive and null for decoration. :100:

File: index.html [hunk 5]
```diff
@@ -88,7 +88,7 @@ <h1 id="primary">Hey There. <br />I'm Lee</h1>
     </section>
     <section id="section-bac">
       <div id="heading-second">
-        <h1 id="heads">My Recent Works</h1>
+        <h2 id="heads">My Recent Works</h2>
         <div id="recent-rule">
```
Comment (leehaney254) [inline index.html:92]: Removed other h1 so that the web page has only one h1. :muscle:

File: index.html [hunk 6]
```diff
@@ -100,7 +100,7 @@ <h1 id="heads">My Recent Works</h1>
         </div>
         <div id="descp-side">
           <div class="description">
-            <p id="head1">Multi-Post Stories</p>
+            <h3 id="head1">Multi-Post Stories</h3>
```
Comment (leehaney254) [inline index.html:103]: Added h3 tags under h2 tags

File: styles.css [hunk 7]
```diff
@@ -3,6 +3,10 @@ body {
   padding: 0;
 }
 
+label {
+  display: none;
+}
+
```
Comment (leehaney254) [inline styles.css:9]: Added labels to form with a display of none. :+1:

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @leehaney254, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - You have submitted a project, that does not have any commits from your coding partner for this milestone. Please note that pair-programming activities require that both you and your partner contribute to the milestone.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

File: index.html [hunk 8]
```diff
@@ -121,7 +121,7 @@ <h3 id="head1">Multi-Post Stories</h3>
         <div id="grid-card">
           <div class="card first-card other-cards">
             <div class="description" id="descp">
-              <h4 class="head-other">Profesional Art Printing Data</h4>
+              <h3 class="head-other">Profesional Art Printing Data</h3>
```
Comment (kamranahmad123) [inline index.html:124]: I changed the header flow from h4 to h3 to structure the valid header flow.

File: index.html [hunk 9]
```diff
@@ -136,7 +136,7 @@ <h4 class="head-other">Profesional Art Printing Data</h4>
           </div>
           <div class="card av-img other-cards">
             <div class="description">
-              <h4 class="head-other">Profesional Art Printing Data</h4>
+              <h3 class="head-other">Profesional Art Printing Data</h3>
```
Comment (kamranahmad123) [inline index.html:139]: I changed the header flow from h4 to h3 to structure the valid header flow/

File: index.html [hunk 10]
```diff
@@ -151,7 +151,7 @@ <h4 class="head-other">Profesional Art Printing Data</h4>
           </div>
           <div class="card teni-img other-cards">
             <div class="description">
-              <h4 class="head-other">Profesional Art Printing Data</h4>
+              <h3 class="head-other">Profesional Art Printing Data</h3>
```
Comment (kamranahmad123) [inline index.html:154]: I changed the header flow from h4 to h3 to structure the valid header flow/

File: index.html [hunk 11]
```diff
@@ -230,7 +230,7 @@ <h2 id="heading">About me</h2>
         <ul id="skill-list" class="flexs">
           <div>
             <li id="rec-side">
-              <span class="skill-tit">Language</span>
+              <h3 class="skill-tit">Language</h3>
```
Comment (kamranahmad123) [inline index.html:233]: The span is changed to h3 for the proper structure flow

File: index.html [hunk 12]
```diff
@@ -241,7 +241,7 @@ <h2 id="heading">About me</h2>
           </div>
           <div>
             <li id="rec">
-              <span class="skill-tit">Frameworks</span>
+              <h3 class="skill-tit">Frameworks</h3>
```
Comment (kamranahmad123) [inline index.html:244]: The span is changed to h3 for the proper structure flow

File: index.html [hunk 13]
```diff
@@ -253,7 +253,7 @@ <h2 id="heading">About me</h2>
           </div>
           <div>
             <li id="circle">
-              <span class="skill-tit">Skills</span>
+              <h3 class="skill-tit">Skills</h3>
```
Comment (kamranahmad123) [inline index.html:256]: The span is changed to h3 for the proper structure flow

File: index.html [hunk 14]
```diff
@@ -57,50 +57,50 @@ <h1 id="primary">Hey There. <br />I'm Lee</h1>
...
         <div id="recent-rule">
           <hr />
         </div>
       </div>
       <div class="card" id="card">
         <div id="img-div">
-          <img id="rec-image" src="./assets/first-card.png" alt="card image" />
-          <img id="large-image" src="./assets/large.png" alt="card image" />
+          <img id="rec-image" src="./assets/first-card.png" alt="" />
+          <img id="large-image" src="./assets/large.png" alt="" />
```
Comment (Olamarx) [inline index.html]: - [ ] You added descriptive alt tags to other img tags and I would believe you forgot to add to these very tags. alt is short form of Alternative Text and it is for reasons when the image can't be accessed for any reason. The alt text would stand in for the image.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @leehaney254, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Webpage fully accessible ✔️ 
- All linters passed ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @leehaney254, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**